README.md

```diff
@@ -13,3 +13,31 @@ You can reach the maintainers of this project at:
 ### Code of conduct
 
 Participation in the Kubernetes community is governed by the [Kubernetes Code of Conduct](code-of-conduct.md).
+
+
+## Product Security Committee (PSC)
+
+The Product Security Committee (PSC) is responsible for triaging and handling the security issues for Kubernetes. Following are the current Product Security Committee:
```

Comment:
> Following are the current Product Security Committee:

Following are the current Product Security Committee members:

or

Following are the current members of the Product Security Committee:

Reply:
Thanks for catching this, fixed!